src/panels/lovelace/badges/hui-summary-badge.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  customElement,
+  html,
+  LitElement,
+  TemplateResult,
+  property,
+} from "lit-element";
+import { computeDomain } from "../../../common/entity/compute_domain";
+import "../../../components/ha-label-badge";
+import { HomeAssistant } from "../../../types";
+import { LovelaceBadge } from "../types";
+import { SummaryBadgeConfig } from "./types";
+
+@customElement("hui-summary-badge")
+export class HuiSummaryBadge extends LitElement implements LovelaceBadge {
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @property() protected _config?: SummaryBadgeConfig;
+
+  public setConfig(config: SummaryBadgeConfig): void {
+    this._config = {
+      icon: "hass:lightbulb-group",
+      domain: "light",
+      state: "on",
+      ...config,
+    };
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass) {
+      return html``;
+    }
+
+    const states: string[] = [];
+
+    for (const entity of Object.keys(this.hass.states)) {
+      if (
+        (!this._config.exclude || !this._config.exclude.includes(entity)) &&
+        computeDomain(entity) === this._config.domain &&
+        this.hass.states[entity].state === this._config.state
+      )
+        states.push(entity);
+    }
+
+    if (states.length === 0 && !this._config.show_always) {
+      return html``;
+    }
+
+    return html`
+      <ha-label-badge
+        .icon=${this._config.icon}
+        .description="${states.length} ${this._config.name ||
+        this._config.domain}${states.length !== 1 ? "s" : ""} ${this._config
```

Comment:
the plural won't work for localization

Reply:
Any suggestions on how to handle? Considering I'm triggering this all off the domain, English is really the only supported language and I'm not really sure how best to localize it.

Reply:
We should just add the plural domain to localize? I guess that could be a lot of domains to add though 🤔

Reply:
Adding this here so its not lost in discord. No idea if this is a way but in Localize if we use Variables for most of is could we potentially do this? 

`{number} {domian}(s) {state}` and then it would be up to the language translation to do the order and change `(s)` to whatever they need it to be? 

The only other solution I can think of is adding plural domains to the localize in the backend?

Reply:
Lokalise just supports plurals, so you can do `{count, plural,\n  one {domain}\n  other {domains}\n}`

Reply:
<https://github.com/home-assistant/frontend/blob/dev/src/translations/en.json#L336-L337>

Reply:
So, I think I'm close, but not sure how to pass the domain as a variable into that function in the translation.